cli/CampaignConfigValidation/CampaignValidator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Cli\CampaignConfigValidation;
```

Comment:
From my point of view, the Validator is more of a domain object and would therefore belong to the `Frontend\BucketTesting\Validation` namespace, but let's discuss that.